src/main/java/edu/wpi/first/nativeutils/vendordeps/VendorDepTask.java

```diff
@@ -41,6 +45,22 @@ public void install() throws IOException {
             getLogger().info("Remotely fetching " + filename);
             downloadRemote(dest);
         }
+
+        var destString = dest.toString();
+        String newFilename;
+        try (BufferedReader reader = Files.newBufferedReader(dest)) {
+            newFilename = new GsonBuilder().create().fromJson(reader, JsonDependency.class).fileName;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        File file = new File(destString);
+        int lastPathSeperator = dest.toString().lastIndexOf('/');
+        boolean didRename = file.renameTo(new File(dest.toString().substring(0, lastPathSeperator + 1) + newFilename));
+        if (didRename == true) {
+          getLogger().info("File Succesfully Renamed");
+        } else {
+          getLogger().warn("Failed to rename file");
```

Comment:
Some information about which file is being renamed and what it was attempted to be renamed to would be helpful